src/SmartComponents/CompletedTaskDetails/Columns.js

```diff
@@ -61,11 +61,60 @@ export const MessageColumn = {
   ),
 };
 
+export const ReportColumn = {
+  title: 'Report',
+  renderExport: (job) => job?.results?.report || '',
```

Comment:
That is what I meant in Slack. We should always expect a `job` to be present here, if it is not, something higher up is wrong.

Reply:
You are right, `job` must be always present and not undefined. The `report` key sometimes though can evaluate to undefined. I have revisited these chainings in the last commit.